resources/assets/coffee/react/beatmapset-page/main.coffee

```diff
@@ -90,6 +90,13 @@ export class Main extends React.Component
     else
       @state.enabledMods
 
+    @setState
+      enabledMods: enabledMods
```

Comment:
`enabledMods` and `currentScoreboardType` should be reset to previous if scoreboard request fails

Reply:
Looks like `currentScoreboardType` not reset to previous state when failed is also a bug in current master.